src/main/java/gmbs/model/ForwardStatusGenerator.java

```diff
@@ -0,0 +1,22 @@
+package gmbs.model;
+
+import java.util.Random;
+
+public class ForwardStatusGenerator {
+
+    Random randomCarStatus = new Random();
+
+    public int randomNumber(int rangeStart, int rangeEnd) {
+        if(rangeStart<0 || rangeEnd<0 || rangeStart>rangeEnd) {
+            throw new IllegalArgumentException();
+        }
+        return randomCarStatus.nextInt(rangeEnd-rangeStart+1)+rangeStart;
+    }
+
+    public boolean isMoveForward(int rangeStart, int rangeEnd, int referenceCondition) {
+        if (referenceCondition<rangeStart || referenceCondition > rangeEnd) {
+            throw new IllegalArgumentException();
+        }
+        return randomNumber(rangeStart,rangeEnd) >= referenceCondition;
+    }
+}
```

Comment:
```suggestion
    public int randomNumber(int rangeStart, int rangeEnd) {
        if (rangeStart < 0 || rangeEnd < 0 || rangeStart > rangeEnd) {
            throw new IllegalArgumentException();
        }
        return randomCarStatus.nextInt(rangeEnd - rangeStart + 1) + rangeStart;
    }

    public boolean isMoveForward(int rangeStart, int rangeEnd, int referenceCondition) {
        if (referenceCondition < rangeStart || referenceCondition > rangeEnd) {
            throw new IllegalArgumentException();
        }
        return randomNumber(rangeStart, rangeEnd) >= referenceCondition;
    }
}
```

저번 리뷰에도 공백 말씀드렸는데요..! 신경써주시면 좋을 것 같습니다.